.drone.yml

```diff
@@ -58,7 +58,7 @@ steps:
       SEVERITY: MEDIUM,HIGH,CRITICAL  --dependency-tree
       FAIL_ON_DETECTION: false
       IGNORE_UNFIXED: false
-      ALLOW_CVE_LIST_FILE: hof-services-config/infrastructure/trivy/.trivyignore.yaml
+      ALLOW_CVE_LIST_FILE: hof-services-config//E_Visa_Contact_Us/
```

Comment:
@DivineTaminangHO we are using trivyignore as global cve file . 
The one you added is also incorrect it has two forward slashes